src/Core/src/Platform/Android/KeyboardManager.cs

```diff
@@ -58,20 +60,17 @@ internal static void ShowKeyboard(this SearchView searchView)
 				return;
 			}
 
-			var textView = searchView.FindViewById(searchViewTextViewId);
+			var queryEditor = searchView.GetFirstChildOfType<EditText>();
```

Comment:
With this change I think we can delete the code above this

```C#
			int searchViewTextViewId = searchView.Resources.GetIdentifier("android:id/search_src_text", null, null);
			if (searchViewTextViewId == 0)
			{
				// Cannot find the resource Id; nothing else to do
				return;
			}
```

because we are no longer searching by id